dojo/settings/settings.dist.py

```diff
@@ -231,6 +231,8 @@
     DD_DELETE_PREVIEW=(bool, True),
     # Feature toggle for new authorization for configurations
     DD_FEATURE_CONFIGURATION_AUTHORIZATION=(bool, True),
+    # List of acceptable file types that can be uploaded to a given object via arbitrary file upload
+    DD_FILE_UPLOAD_TYPES=(list, ['.txt', '.pdf', '.json', '.xml', '.yml', '.png', '.jpeg']),
```

Comment:
There are quite a few other file extensions in the unittests folder, for example:

.csv, .html, .sarif, .xslx, .html, .js, .nessus, .zip

Reply:
Good suggestion